### ISSUE TITLE
events?

### ISSUE THREAD

**Issue by pehrlich** [jettro/c3-angular-directive#7]
From what I can tell, none of the `data.` events are available - such as onclick, onmouseover, etc. Is that correct?

**Comment by jettro** [comment]
Can you explain a bit more what you mean, or what you try to accomplish?

**Comment by pehrlich** [comment]
So here's a couple of commits I've hacked together which demonstrate something:

Add mouseover/mouseout: https://github.com/pehrlich/c3-angular-sample/commit/279b2a8a2d44a722cf577d4977af577c0e62d92f .  This is functional and follows established patterns, but I don't like the idea of having to explicitly mirror every API feature. Perhaps there's a better and more meta way of removing all this non-DRY stuff, while reducing maintainece reqs. https://github.com/pehrlich/c3-angular-sample/blob/master/c3js-directive.js#L24-L71

I found that direct access to the chart was pretty important, for experimenting with their raw api, and calling their methods such as `tooltip.show`. So not finding a better way, I exposed it in this method: https://github.com/pehrlich/c3-angular-sample/commit/aca1179fd04e8d97ea18cf262b9d7a9f344cf651
